install/amazon_rds/_01_public.mdx

```diff
@@ -27,13 +24,11 @@ In your [AWS Console](https://console.aws.amazon.com/rds/), modify your existing
 
 Then set the following configuration parameters:
 
-<ImgRdsParamGroup2 />
-
  Name | New Value | &nbsp;
  ---- | ------ | -------
  [pg\_stat\_statements.track](https://www.postgresql.org/docs/current/pgstatstatements.html#id-1.11.7.39.9) | ALL | Optional, enables tracking of queries inside stored procedures
 [shared\_preload\_libraries](https://www.postgresql.org/docs/current/runtime-config-client.html#RUNTIME-CONFIG-CLIENT-PRELOAD) | pg\_stat\_statements
-[track\_activity\_query\_size](https://www.postgresql.org/docs/current/runtime-config-statistics.html#RUNTIME-CONFIG-STATISTICS-COLLECTOR) | 2048
+ [track\_io\_timing](https://www.postgresql.org/docs/current/runtime-config-statistics.html#RUNTIME-CONFIG-STATISTICS-COLLECTOR) | on | Optional, enables tracking of per-query I/O statistics
```

Comment:
In aws management console, I see it as `0, 1` value, but should it be `1` instead?

![image](https://user-images.githubusercontent.com/911433/191381250-6a31d948-184e-4dbb-9c90-c293b5672121.png)